docs/GETTING_STARTED.md

```diff
@@ -132,15 +132,16 @@ does not have access to.
 
 ### Creating the SOCI index
 
-To create the index run:
+To create the index, which can be pushed to ORAS-compatible registry, run:
 ```
-sudo ./soci create localhost:5000/rabbitmq:latest
+sudo ./soci create --oras localhost:5000/rabbitmq:latest
 ```
-Behind the scenes SOCI is interacting with containerd.  So the tag
-`localhost:5000/rabbitmq` has to exist in containerd - that is what SOCI is
-referencing.  This created two kinds of objects.  The first is a series of ztocs
-(one per layer).  A ztoc is a table of contents for compressed data (z is the
-compression, toc is Table Of Contents).  The other is a manifest that relates
+The `--oras` flag instructs the CLI to build an ORAS-compatible manifest, 
```

Comment:
I think this could still use another sentence to explain why this matters at all. 

> The community is in the process of migrating from ORAS to the standardized OCI Artifact Manifest spec, but support for the latter in libraries and reference registries is not yet available. Until that work is complete, SOCI can be used with ORAS tooling using the `--oras` flag.

I'm not sure that's totally right, but something like that.

Reply:
Maybe go with the following text?

```
The `--oras` flag instructs the CLI to build an ORAS-compatible manifest, 
as opposed to an OCI Artifact Manifest. By default, the CLI will build the latter.
Since community is in process of migrating from ORAS to OCI Artifact Manifest, 
there's no available registry supporting latter. We suggest using ORAS registry along
with `--oras` flag to be able to push the SOCI index to registry.
```
```

Reply:
I think Kern's text was fine.

Reply:
Updated the text.